uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -799,10 +799,20 @@ sub dicom_to_minc {
     my ($d2m_cmd,$d2m_log,$exit_code);
     my $message = '';
     my $upload_id = getUploadIDUsingTarchiveSrcLoc($tarchive_srcloc);
+
+    # create the excluded series description regex necessary to exclude the
+    # series description specified in the Config Setting
+    # excluded_series_description
+    my $excluded_regex;
+    foreach my $excluded_series_desc (@$exclude) {
```

Comment:
The computation of `$exclude_regex` can be done in one line:

`my $excluded_regex = join('|', @$exclude);`

(and no need to `chop` it)